website/src/app/portal/components/pro-blocks/page-headers/page-header-4.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+
+import {
+	Breadcrumb,
+	BreadcrumbItem,
+	BreadcrumbLink,
+	BreadcrumbList,
+	BreadcrumbSeparator,
+} from '@/app/portal/components/ui/breadcrumb';
+
+import { usePathname } from 'next/navigation';
+
+export function PageHeader4() {
+	const pathname = usePathname();
+	const segments = pathname.split('/').filter(Boolean);
+
+	return (
+		<div className="container mx-auto flex flex-col py-9">
+			<Breadcrumb>
+				<BreadcrumbList>
+					<BreadcrumbItem>
+						<BreadcrumbLink href="/">Home</BreadcrumbLink>
+					</BreadcrumbItem>
+					{segments.map((segment, index) => {
+						const href = '/' + segments.slice(0, index + 1).join('/');
+						const isLast = index === segments.length - 1;
+
+						return (
+							<>
```

Comment:
Each child in a list should have a unique "key" prop.

Check the render method of `ol`. It was passed a child from PageHeader4. See https://react.dev/link/warning-keys for more information.